src/shared/contexts/query.tsx

```diff
@@ -192,7 +129,9 @@ export const simplify = (text, vars = {}) => {
     )
       .map(node => node.property.name)
       .reduce((acc, curr) => {
-        acc[curr] = vars[curr]
+        if (vars[curr]) {
+          acc[curr] = vars[curr]
+        }
```

Comment:
this code was incorrect.

What it was doing:
*detecting a used variable (`v.myVar`) in the query string
* then expected that it must be defined externally in the vars function parameter
* However, if it's defined in the query code itself, and not the vars, it was injecting into the flux `option v = {myVar: undefined}`

Reply:
This code was left in. Even though it's wrong, per the explanation^^ above.
But I'm leaving notebooks code alone for now, which means leaving the shared QueryContext alone.